src/main/resources/application.yaml

```diff
@@ -50,6 +50,8 @@ quarkus:
         level: DEBUG
       "org.apache.kafka.clients.Metadata":
         level: DEBUG
+      "io.fabric8.kubernetes.client":
+        level: DEBUG
```

Comment:
We should use an env here. This is great for troubleshooting stuff like ECONNRESET but we should probably default to info.

Reply:
Let's do a single-arch build with DEBUG, so we don't have to update lots of yaml.